test/screenshot/select/index.tsx

```diff
@@ -37,23 +62,9 @@ class SelectTest extends React.Component {
   }
 }
 
-const variants = [
-  {},
-  {box: true},
-  {outlined: true},
-];
-
-const rtlMap = [
-  {},
-  {isRtl: true},
-];
-
-const disabledMap = [
-  {},
-  {value: 'pomsky'},
-  {disabled: true},
-];
-
+const variants = [{}, {box: true}, {outlined: true}];
```

Comment:
This is a bit hard to follow. 

The first 3 arrays of objects, as is, are not maps themselves. Should they be renamed?
Where is variantKey coming from? 
Are any of the values being used (e.g. true or 'pomsky'?)

Reply:
ya it is a bit hacked together. I wanted to programatically add:

```
<Select box ... />
<Select outlined ... />
```

in an array format.

Reply:
variantKey is either `''`, `box`, or `outlined`

Reply:
I updated the screenshot and moved the {value: pomsky} out, which in turn created a different screenshot. But I the `maps` make more sense.